.github/workflows/image-publish-dockerhub.yaml

```diff
@@ -0,0 +1,52 @@
+name: Publish Docker Image to docker hub
```

Comment:
I am not sure we want a manually triggered workflow for this, why isn't the current release workflow sufficient?

Reply:
Besides the release workflow, ECR image could also be published by '[docker-image-publish.yaml'](https://github.com/stargate/jsonapi/blob/main/.github/workflows/docker-image-publish.yaml). 
I was thinking about making the similar one to dockerhub image also.

Reply:
Deleted.